molecule/provisioner/lint/ansible_lint.py

```diff
@@ -127,7 +79,7 @@ def bake(self):
             options,
             exclude_args,
             sum(x_args, ()),
-            self._config.provisioner.playbooks.converge,
+            self._playbook,
```

Comment:
Sorry, struggling to understand why this has changed now, could you explain? Thanks.

Reply:
Because it's been converted to a shorter property below: https://github.com/ansible/molecule/pull/1714/files/b2c61c3319f38a9afbfa80e831ff0b32eaed2080#diff-7e1ab4797a3dc90b904bce62fa037fe3R159

Reply:
Yeah, and since this is the mixin it will either be the converge or the verify playbook